api/layer/container.go

```diff
@@ -169,16 +169,11 @@ func (n *layer) createContainer(ctx context.Context, p *CreateBucketParams) (*ci
 	return bktInfo.CID, nil
 }
 
-func (n *layer) setContainerEACLTable(ctx context.Context, cid *cid.ID, table *eacl.Table) error {
+func (n *layer) setContainerEACLTable(ctx context.Context, cid *cid.ID, table *eacl.Table, sessionToken *session.Token) error {
 	table.SetCID(cid)
+	table.SetSessionToken(sessionToken)
```

Comment:
It seems we can accept only `*eacl.Table` that already has session token inside

Reply:
I decided to require session token as a separated argument to avoid forgetting to attach the token to `eacl.Table`  before calling of `setContainerEACLTable` in case of possible further use. 
But it's just a matter of taste.